runtime/lua/vim/treesitter/highlighter.lua

```diff
@@ -4,12 +4,30 @@ local Range = require('vim.treesitter._range')
 
 local ns = api.nvim_create_namespace('treesitter/highlighter')
 
----@alias vim.treesitter.highlighter.Iter fun(end_line: integer|nil): integer, TSNode, vim.treesitter.query.TSMetadata, TSQueryMatch
+--- @class vim.treesitter.highlighter.Info
+--- @inlinedoc
+--- The range of the highlight
+--- @field range Range6
+--- The group of the highlight
+--- @field hl_group string
+--- The priority of the highlight
```

Comment:
are there relevant docs we can link to, for some of these fields? the current descriptions are almost useless

```suggestion
--- Highlight priority |treesitter-highlight-priority|
```